src/migrations/20191118182245-create-booking.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('bookings', {
```

Comment:
I think this does not contain enough data to define a `booking`, I feel `roomId` is missing here?

Reply:
we added room id to booking table